Revision Tests/Bohdanov_revision.md

```diff
@@ -0,0 +1,31 @@
+## Please translate into English
+
+**_1. Если бы я был умнее (по жизни) то я бы сделал вчера бэкап всей базы и только потом применял свой ПР к коду на проде. Жаль что я не умный как мой папа._**
+
+> If I had been smarter(in life), I would have backed up the whole base and only after it applied my pull request to the code on the prod. I wish I was smart as my dad.
+
+**_2. Если бы все студенты читали теорию и только потом писали перевод анализируя предложения, то прогресс бы был очевиден. Жаль что все хотят выучить английский брутфорсом, минуя ранние этапы обучения и анализа и отработки навыков до автоматизма. Я бы хотел чтобы вы не торопились._**
+
+> If all students read a theory and only then wrote a translation by analyzing the sentences, the progress would be obvious. If only they wouldn't tried to learn english by brutforcing, bypassing the early stages of learning and practicing skills to automation. I want you to not hurry up.
+
+**_3. Если бы сегодня мне не нужно было садиться за руль, то вчера бы я знатно набрался с друзьями в ближайшем баре и орал бы в микрофон свою любимую песню. Жаль что автопилот не является повсеместной функцией во всех автомобилях, это могло бы значительно разнообразить досуг водителей._**
+
+> If I didn't have to drive today, yesterday I would have got (gotten?) drunk with my friends in the nearest bar and screamed in the mic my favorite song.I wish an autopilot was generic function in all cars, it could significantly diversify drivers free time. 
+
+**_4. Если бы она проверяла работоспособность билда после слияния своих коммитов с мастером в прошлом, то сейчас бы у нее не было бы общения с клиентом по поводу даунтайма на проекте и финансовых потерях в целом._**
+
+> If she had checked a build working capacity after merging her commits to the master in the past, then right now she wouldn't have a conversation with the client regarding the project downtime and financial lost overall.
+
+**_5. - Жаль что у тебя всего лишь одна бутылка виски. Так бы я могла остаться подольше._**
+**_- Да, ты права, и мне жаль, что у меня не две бутылки виски._**
+
+> 
+> If only you had more than 1 bottle of whiskey. In that case I could stay longer. Yeah, you are right, I wish I had 2 bottles of whiskey too.
+
+**_6. Если бы он спал 7-8 часов в сутки, то и чувствовал бы себя сегодня бодрее и небыло бы проблем с настроением и агрессией. Жаль что он играет по 6 часво после работы._**
+
+> If he had slept 7-8 hours in the day, he would feel himself much cheerful and wouldn't have a problem with mood and aggression. If only he hadn't played for 6 hours after work.
```

Comment:
If he slept (in general)
he would feel much berre (why adding himself)
much more cheerful
If only he didn't play (on a regular basis)